app/main/menu.js

```diff
@@ -441,44 +464,58 @@ class AppMenu {
 			}, {
 				type: 'separator'
 			}, {
+				label: TranslationUtil.__('Minimize'),
+				role: 'minimize'
+			}, {
+				label: TranslationUtil.__('Close'),
+				role: 'close'
+			}, {
+				label: TranslationUtil.__('Quit'),
 				role: 'quit',
 				accelerator: 'Ctrl+Q'
 			}]
 		}, {
-			label: '&Edit',
+			label: TranslationUtil.__('Edit'),
 			submenu: [{
+				label: TranslationUtil.__('Undo'),
 				role: 'undo'
 			}, {
+				label: TranslationUtil.__('Redo'),
 				role: 'redo'
 			}, {
 				type: 'separator'
 			}, {
+				label: TranslationUtil.__('Cut'),
 				role: 'cut'
 			}, {
+				label: TranslationUtil.__('Copy'),
 				role: 'copy'
 			}, {
+				label: TranslationUtil.__('Paste'),
 				role: 'paste'
 			}, {
+				label: TranslationUtil.__('Paste and Match Style'),
 				role: 'pasteandmatchstyle'
 			}, {
 				type: 'separator'
 			}, {
+				label: TranslationUtil.__('Select All'),
 				role: 'selectall'
 			}]
 		}, {
-			label: '&View',
```

Comment:
Why did we have `&` here for?

Reply:
Not sure. Maybe @akashnimare can help out here.

Reply:
This came from a standard electron template. Not sure if we need this now.